services/divshot/README.md

```diff
@@ -0,0 +1,112 @@
+Divshot
+========
+
+Our static site service of choice
+
+How to set up Divshot and CircleCI for a new project
+----------------------------------------------------
+
+### Divshot.io Actions
+
+First thing to do is set up a new Divshot app via the [divshot.io](http://divshot.io) dashboard.
+
+![](http://files-misc.s3.amazonaws.com/create-app.png)
+
+Once you've set up the app with divshot, go to the CLI and `divshot init` an app with the same name.
+
+Install the [divshot-cli](https://github.com/divshot/divshot-cli#install) by running:
+
+`npm install divshot-cli -g`
+
+Running `divshot init` will generate a new file called `divshot.json` in the root of your project. It should look something like this:
+
+`divshot.json` example file:
+```json
+{
+  "name": "project-name",
+  "root": "dist",
+  "clean_urls": true,
+  "error_page": "error.html"
+}
+```
+
+To kick off the first deploy, type `divshot push production` to push to divshot's production environment. `divshot push` will by default push to the Development environment.
+
+### Set up some authentication 
+
+[Authentication with Divshot](http://docs.divshot.com/guides/http-authentication) is a snap.
```

Comment:
Maybe add something about _Add the creds to 1password_ linking to https://github.com/sparkbox/how_to/tree/master/software/1password